docs/snapcraft-advanced-features.md

```diff
@@ -171,10 +171,9 @@ thoughts on services and their security, visit the
 
 ### Limiting the number of installed files
 
-To check the list of files included in your snap, you can use `dpkg -c` on
```

Comment:
Are we completely losing this functionality with the move to squashfs? Do we have anything equivalent? It was quite handy to be able to open the .snap in the Archive Manager, or use `dpkg -c`.